### PR TITLE
Archives

### PR DESCRIPTION
**Pull request by jmbarne3**
**Description**
Added archive template and `news-archive` single page template to handle redirect.

**Motivation and Context**
To replace the current news-archive logic and page.

**How Has This Been Tested?**
Currently in development.

** To Do **
We need a solution for the resources menu so this isn't hard coded into a template.

**Types of changes**

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

**Checklist:**

- [x] My code follows the code style of this project.
- [x] My change requires an update to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: archive.php [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+				<?php ucfwp_the_posts_pagination(); ?>
+			<?php else : ?>
+				<div class="alert alert-info">
+					<p>There are currently no stories for <?php echo date( 'F Y' ); ?>.</p>
+				</div>
+			<?php endif; ?>
+		</div>
+		<div class="col-md-4">
+		<h2 class="h6 text-uppercase text-default-aw mb-4">Resources</h2>
+		<ul class="list-unstyled">
```
Comment (cadieb) [inline archive.php]: We could use `wp_nav_menu` to output the Resources menu, it's used for the Tag template page:
https://github.com/UCF/Today-Child-Theme/blob/rc-v1.0.0/tag.php#L61-L66

Comment (jmbarne3) [reply archive.php]: Added

File: includes/archive-functions.php [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+
+		$posts = new WP_Query( $args );
+		$next_have_posts = $posts->post_count > 0 ? true : false;
+
+		ob_start();
+	?>
+		<div class="row justify-content-around mt-3 mb-5">
+			<?php if ( $prev_have_posts ) : ?>
+			<div class="col">
+				<a href="<?php echo $prev_url; ?>" class="btn btn-primary"><span class="fa fa-arrow-left"></span> <?php echo $prev_month->format( 'M Y' ); ?></a>
```
Comment (cjg89) [inline includes/archive-functions.php]: Icon `<span>` could use `aria-hidden="true"` attribute

File: includes/archive-functions.php [hunk 3]
```diff
@@ -0,0 +1,59 @@
...
+			</div>
+			<?php endif; ?>
+			<?php if ( $next_have_posts ) : ?>
+			<div class="col text-right">
+				<a href="<?php echo $next_url; ?>" class="btn btn-primary"><?php echo $next_month->format( 'M Y' ); ?> <span class="fa fa-arrow-right"></span> </a>
```
Comment (cjg89) [inline includes/archive-functions.php]: Icon `<span>` could use `aria-hidden="true"` attribute

File: gulp-config.template.json [hunk 4]
```diff
@@ -1,4 +0,0 @@
-{
```
Comment (cjg89) [inline gulp-config.template.json]: Was this file deleted intentionally?

File: includes/archive-functions.php [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+			</div>
+			<?php endif; ?>
+			<hr>
```
Comment (cjg89) [inline includes/archive-functions.php]: This `<hr>` isn't currently visible on the page (probably due to it being within a `.row`)?  Consider removing it for now

**Pull request merged**